examples/demo13.py

```diff
@@ -0,0 +1,378 @@
+    SCA_cent_pos = wcs.toWorld(galsim.PositionD(wfirst.n_pix/2, wfirst.n_pix/2))
+
+    # We are going to just randomly distribute points in (X, Y).
+    # If we had a real galaxy catalog with positions in terms of RA, dec we could use wcs.toImage()
+    # to find where those objects should be in terms of (X, Y).
+    pos_rng = galsim.UniformDeviate(random_seed)
+    # Make a list of (X, Y) values, eliminating the 10% of the edge pixels as the object centroids.
+    x_stamp = []
+    y_stamp = []
+    for i_gal in xrange(n_tot):
+        x_stamp.append(pos_rng()*0.8*wfirst.n_pix + 0.1*wfirst.n_pix)
+        y_stamp.append(pos_rng()*0.8*wfirst.n_pix + 0.1*wfirst.n_pix)
+        # Note that we could use wcs.toWorld() to get the (RA, dec) for these (x, y) positions.  Or,
+        # if we had started with (RA, dec) positions, we could have used wcs.toImage() to get the
+        # CCD coordinates for those positions.
+
+    # Make the 2-component parametric GSObjects for each object, including chromaticity (roughly
+    # appropriate SEDs per galaxy component, at the appropriate galaxy redshift).  Note that since
+    # the PSF is position-independent within the SCA, we can simply do the convolution with that PSF
+    # now instead of using a different one for each position.
+    gal_list = []
+    for i_gal in xrange(n_use):
+        logger.info('Processing the object at row %d in the input catalog.'%i_gal)
+
+        # Use built-in routines for constructing COSMOS objects:
+        gal = galsim.makeCOSMOSObj(cat, i_gal, chromatic=True)
+
+        # Convolve the chromatic galaxy and the chromatic PSF
+        final = galsim.Convolve(gal, PSF)
+        logger.debug('Pre-processing for galaxy %d completed.'%i_gal)
+        gal_list.append(final)
+
+    # Calculate the sky level for each filter, and draw the PSF and the galaxies through the
+    # filters.
+    for filter_name, filter_ in filters.iteritems():
+        logger.info('Beginning work for {0}.'.format(filter_name))
+
+        # Drawing PSF.  Note that the PSF object intrinsically has a flat SED, so if we
+        # convolve it with a galaxy, it will properly take on the SED of the galaxy.  However,
+        # this does mean that the PSF image being drawn here is not quite the right PSF for
+        # the galaxy.  Indeed, the PSF for the galaxy effectively varies within it, since it
+        # differs for the bulge and the disk.  To make a real image, one would have to choose SEDs
+        # for stars and convolve with a star that has a reasonable SED, but we are just going to
+        # draw with a flat SED for this demo.
+        out_filename = os.path.join(outpath, 'demo13_PSF_{0}.fits'.format(filter_name))
+        img_psf = galsim.ImageF(64,64)
+        PSF.drawImage(bandpass=filter_, image=img_psf, scale=wfirst.pixel_scale)
+        # Artificially normalize to a total flux of 1 for display purposes.
+        img_psf /= img_psf.array.sum()
```

Comment:
I'm not sure if this is the best demo of how to draw a PSF.  PSFs should have unit flux.  So it's one thing for the SED of the PSF to be flat, but it's also not normalized correctly for this drawing to make sense.  So rather than divide by the sum to get back to unit flux, I think it would be better to show how to get the flux to 1 in the first place.
```
# Approximate a point source.
point = galsim.Gaussian(sigma=1.e-8, flux = 1)
# Use a flat SED here, but could use something else.  A stellar SED for instance.  
# Or a typical galaxy SED.  Depending on your purpose for drawing the PSF.
star_sed = galsim.SED(lambda x:1).withFlux(1.,filter_)  # Give it unit flux.
star = galsim.Convolve(point * star_sed, PSF)   
star.drawImage(bandpass=filter_, image=img_psf, scale=wfirst.pixel_scale)
```